web/panorama/panorama/tests/test_api_adjacent.py

```diff
@@ -5,47 +5,21 @@
 class AdjacentPanoramaApiTest(PanoramaApiTest):
 
     def test_get_base_panorama(self):
-        response = self.client.get('/panorama/opnamelocatie/PANO_1_2014/')
+        response = self.client.get('/panorama/panoramas/PANO_1_2014/')
         self.assertEqual(response.status_code, 200)
         self.assertIn('pano_id', response.data)
         self.assertEqual(response.data['pano_id'], 'PANO_1_2014')
```

Comment:
adjacent functionaliteit lijkt niet meer getest te worden ?

Reply:
Het opnamelocatie endpoint is komen te vervallen, en het panoramas endpoint doet dat niet meer rechtstreeks.